components/RuiButton/src/RuiButton.ts

```diff
@@ -83,6 +89,20 @@ export class RuiButton extends LitElement {
 
 	/* #region Methods */
 
```

Comment:
Can you also please add

```
protected createRenderRoot() {
		return this.attachShadow({mode: 'open', delegatesFocus: true});
}
```

Reply:
yep can do. that said delegatesFocus is very poorly supported right now.